ccm

```diff
@@ -24,22 +24,12 @@ import pkg_resources
 from six import print_
 
 from ccmlib import common
-from ccmlib.cmds import cluster_cmds, command, node_cmds
+from ccmlib.cmds import cluster_cmds, node_cmds
+from ccmlib.cmds.common import get_command
 from ccmlib.remote import (PARAMIKO_IS_AVAILABLE, execute_ccm_remotely,
                            get_remote_options, get_remote_usage)
 
 
-def get_command(kind, cmd):
```

Comment:
does this break compatibility ?  would any client be calling this ? (any client that cannot easily add `from ccmlib.cmds.common import get_command` ?) 

if so, we should put in (something like) a deprecated
```
def get_command(kind, cmd):
    import ccmlib.cmds.common
    return common.get_command(kind, cmd)
```

(`common.get_version_from_build` has a deprecated coding example, not sure what's idiomatic here)

Reply:
I don't think so:
-  `ccm` is executable` no one should use it, when you import a method it becomes and there is no way
- when you import anything it becomes part of the package and you can import it as well.
Example:
ccmlib.cmds.command.py imports `PARAMIKO_IS_AVAILABLE` from `ccmlib.remote`
But you can import `PARAMIKO_IS_AVAILABLE` from `ccmlib.cmds.command` too:
```
from ccmlib.cmds.command import PARAMIKO_IS_AVAILABLE
```

Reply:
if folk cannot import `ccm.get_command` then we're good.